### PR TITLE
`dl-v2`  todos

### PR DESCRIPTION
**Pull request by aniketsinghrawat**

### CODE REVIEW

File: weather_dl_v2/cli/app/cli_config.py [hunk 1]
```diff
@@ -50,6 +50,8 @@ def from_dict(cls, config: t.Dict):
 
 def get_config():
     global cli_config
+    # TODO: Update this so cli can work from any folder level.
+    # Right now it only works in folder where cli_config.json is present
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/cli_config.py:54]: nit: add `.`.

File: weather_dl_v2/cli/app/utils.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def wrap_func(*args, **kwargs):
 
     return wrap_func
 
-
+# TODO: Return raw response if a seperate -j/--json flag is passed.
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/utils.py:41]: nit: `TODO: Add a flag (may be -j/--json) to support raw response.`

File: weather_dl_v2/fastapi-server/main.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ async def lifespan(app: FastAPI):
     os.makedirs(os.path.join(os.getcwd(), "config_files"), exist_ok=True)
     # Retrieve license information & create license deployment if needed.
     await create_pending_license_deployments()
-
+    # TODO: Automatically create required indexes on collections on server startup.
```
Comment (mahrsee1997) [inline weather_dl_v2/fastapi-server/main.py:56]: nit: `# TODO: Automatically create required indexes on firestore collections on server startup.`

File: weather_dl_v2/fastapi-server/routers/license.py
Comment (mahrsee1997) [inline weather_dl_v2/fastapi-server/routers/license.py:1]: Remove unnecessary TODOs.
eg: # TODO: Make use of google secret manager.

File: weather_dl_v2/cli/app/subcommands/download.py [hunk 4]
```diff
@@ -51,7 +51,7 @@ def get_downloads(
 
     print(as_table(download_service._list_all_downloads()))
 
-
+# TODO: Add support for selecting multiple configs using *.cfg notation.
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/download.py:54]: nit:  `selecting` -> `submitting` .

File: weather_dl_v2/cli/app/cli_config.py (hunk 1 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/cli_config.py]: updated in [ad2d386](https://github.com/google/weather-tools/pull/407/commits/ad2d3869472a5410ba6dda75dec46eb217cbf6f7)

File: weather_dl_v2/fastapi-server/main.py (hunk 3 above)
Comment (aniketinfocusp) [reply weather_dl_v2/fastapi-server/main.py]: [ad2d386](https://github.com/google/weather-tools/pull/407/commits/ad2d3869472a5410ba6dda75dec46eb217cbf6f7)

File: weather_dl_v2/fastapi-server/routers/license.py
Comment (aniketinfocusp) [reply weather_dl_v2/fastapi-server/routers/license.py:1]: updated in [8ebe1cf](https://github.com/google/weather-tools/pull/407/commits/8ebe1cf6a0be6fbfaf5dde85258bb56bea4c953e)

File: weather_dl_v2/cli/app/subcommands/download.py (hunk 4 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/subcommands/download.py]: [fb2bf50](https://github.com/google/weather-tools/pull/407/commits/fb2bf50f8b888fe052a655ddb0a93becb48441a6)

**Pull request merged**